db/test/fns/queue_test.js

```diff
@@ -125,14 +125,39 @@ suite(testing.suiteName(), function() {
         await db.fns.ensure_task_group('0cM7dCL2Rpaz0wdnDG4LLg', 'sched', expires);
         await db.fns.ensure_task_group('jcy-h6_7SFuRuKLPByiFTg', 'sched', expires);
 
-        const tgs = await db.fns.get_task_group('jcy-h6_7SFuRuKLPByiFTg');
+        const tgs = await db.fns.get_task_group2('jcy-h6_7SFuRuKLPByiFTg');
```

Comment:
i don't think this needs to change (see above - we should be free to change the inputs/outputs of our database functions as they are not exposed as a data interface directly.

Reply:
Unfortunately we can't. I've written in the comment above. 
See: [db docs](https://github.com/taskcluster/taskcluster/blob/main/db/README.md#stored-functions) and [db best practices](https://github.com/taskcluster/taskcluster/blob/main/dev-docs/best-practices/db.md#redefining-db-functions)